website/docs/admin_server_updating_pipeline.md

```diff
@@ -0,0 +1,33 @@
+---
+id: admin_server_updating_pipeline
+title: Updating Pipeline to latest Release
+sidebar_label: Updating Pipeline
+description: Information for admins about updating the pipeline release.
+toc_max_heading_level: 5
+---
+
+:::tip Pipeline releases
+Pipeline releases are official bundles with the latest tested and stable addons, launchers and their dependencies for your pipeline.
+:::
+
+The `Update Pipeline` tool is designed to be user-friendly, transparent, and fast. It automatically detects which addons need updates and identifies the platforms you’re already using (whether it’s MacOS, Windows, or Linux). The update process only requires two simple clicks, select the menu option, and hit the `Update Pipeline` button.
+
+:::info
+Keep in mind This will not affect your existing setup, a new bundle will be created for the new release.
```

Comment:
I feel there is a better word than "setup". Maybe "current production configuration"?.

Also small typo on "This" needs lowercase "this".

Reply:
Oh thank you. yeah, "current production configuration" sounds better. 
I've added it in https://github.com/ynput/ayon-documentation/pull/266/commits/53e99d4b6d0a9c59e8a9b736dd73176f15172978